templates/rc.local.erb

```diff
@@ -61,6 +61,15 @@ The password can be found in /var/local/password as well as this screen.
 To set a new password, run the command: reset_password.rb
 " > /etc/issue
 
+# Regenerate network script for legacy CLI commands
+rm /etc/sysconfig/network-scripts/ifcfg-enp*
+echo \
+"
+DEVICE=$PRIMARY_INTERFACE
```

Comment:
The nework doesn't depend on the ifcfg scripts. They're just there for legacy support. I deleted it and rebooted the VM and it worked just fine.